package.json

```diff
@@ -29,7 +34,10 @@
   ],
   "dependencies": {
     "Base64": "0.3.0",
+    "es6-promise": "^4.2.5",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
For future-proofing (especially since this library will eventually be used in React Native), we should consider a library that does not modify the global scope, such as https://github.com/lquixada/cross-fetch

See:
https://github.com/matthew-andrews/isomorphic-fetch/issues/125
https://github.com/apollographql/apollo-fetch/issues/22

Reply:
I am good with using cross-fetch instead, @brettritter-okta what do you think?

Reply:
cross-fetch looks good to me!